oaknational/Oak-Web-Application
# [CUR-640] Updated `ip` dependency

## Description

- Updated `ip` as per https://github.com/oaknational/Oak-Web-Application/security/dependabot/90

## Issue(s)

`CUR-640`

## How to test
Generally test app/tests work as expected, minor change to private IP tests, see https://github.com/indutny/node-ip/commit/32f468f1245574785ec080705737a579be1223aa
